src/xwayland/Server.cpp

```diff
@@ -203,19 +225,18 @@ bool CXWaylandServer::tryOpenSockets() {
 
         uint64_t pid = 0;
         try {
-            pid = std::stoi(std::string{pidstr, 11});
+            pid = std::stoul(pidstr);
```

Comment:
ull? isnt pid a u64 on modern systems?

Reply:
`pid_t` in linux is an `int` 
 https://ftp.gnu.org/old-gnu/Manuals/glibc-2.2.3/html_node/libc_554.html

Reply:
my bad